extension/src/plots/webview/messages.ts

```diff
@@ -433,6 +438,25 @@ export class WebviewMessages {
     writeFile(file.path, plotId)
   }
 
+  private async exportPlotAsSvg(svg: string) {
+    const file = await showSaveDialog(
+      join(this.dvcRoot, 'visualization.svg'),
```

Comment:
[F] I lost a couple of SVGs because they were saved into a folder that VS Code couldn't open. Personally I think this is a better default.

Reply:
Do we want to give the file a unique name (the title of the plot maybe)?

Reply:
I recreated exactly what vega does but I can update.